tests/lib/bash/test_header

```diff
@@ -105,25 +105,6 @@
 #         set CYLC_TEST_HOST and CYLC_TEST_OWNER for remote job tests.
 #     port_is_busy $PORT
 #         Return 0 if $PORT is busy or 1 if $PORT is not busy.
-#     cylc_ws_init $NS $UTIL
```

Comment:
We can also now get rid of the ``port_is_busy`` method, which was only used across the test suite inside this ``cylc_ws_init`` method. (And now I have noticed it, we should drop the ``# tags`` on new line 580, just above where that method is defined, which was clearly for the aid of some developer who neglected to remove it. From ``git blame`` I can see that the developer in question was in fact myself :grimacing:.)

Reply:
Good spot, thanks for reviewing it so quickly. Updating PR in a few moments. Thanks @sadielbartholomew !

Reply:
Hopefully removed the right lines [here](https://github.com/cylc/cylc/pull/3024/commits/860b1b131ab1456b4df3bd7409a16037f6050d45)

Reply:
Excellent, thanks. Given the time in NZ I imagine you are no longer online so I have given two Travis CI jobs that were failing on flaky tests a kick. Once they pass I'll get this merged.